packages/twenty-server/src/modules/workflow/workflow-executor/types/workflow-step-settings.type.ts

```diff
@@ -16,7 +17,7 @@ type BaseWorkflowStepSettings = {
 export type WorkflowCodeStepInput = {
   serverlessFunctionId: string;
   serverlessFunctionVersion: string;
-  payloadInput: object;
+  [key: string]: any;
```

Comment:
I personally really prefer to distinguish payload from `serverlessFunctionId` and `serverlessFunctionVersion`